Gemfile

```diff
@@ -2,13 +2,13 @@
 
 source 'https://rubygems.org'
 
-gem 'pry', '~> 0.12.2'
+gem 'pry', '>= 0.13'
 gem 'rake'
 gem 'rspec', '~> 3.8'
 gem 'rubocop', '0.63.1'
 
 platform :mri do
-  gem 'pry-byebug'
+  gem 'pry-byebug', "~> 3.10.1"
```

Comment:
I had to update them as I was getting the following error when running specs:
> undefined method `=~' for class `Pry::Code'

Related: https://gitlab.com/gitlab-org/customers-gitlab-com/-/issues/6143